torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -1300,3 +1305,138 @@ AOTITorchError aoti_torch_cpu__weight_int4pack_mm_cpu_tensor(
     *ret0 = new_tensor_handle(std::move(tmp_result));
   });
 }
+
+class StableIValueConverter : public c10::OperatorKernel {
+ public:
+  StableIValueConverter(void (*fn)(StableIValue*, int64_t, int64_t))
+      : fn_(fn) {}
+
+  void operator()(
+      const c10::OperatorHandle& op,
+      c10::DispatchKeySet keyset,
+      torch::jit::Stack* stack) {
+    const auto& schema = op.schema();
+    const auto num_returns = schema.returns().size();
+    const auto num_arguments = schema.arguments().size();
+
+    StableIValue* ministack = (StableIValue*)malloc(
+        (num_arguments + num_returns) * sizeof(StableIValue));
```

Comment:
nit: Why not std::array? Looks like we're still in C++ land, this buys us some safety (from not needing to free or an exception happening in the middle)

Reply:
Using std::vector